src/execution/physical_plan/plan_window.cpp

```diff
@@ -54,7 +76,12 @@ unique_ptr<PhysicalOperator> PhysicalPlanGenerator::CreatePlan(LogicalWindow &op
 		}
 
 		// Chain the new window operator on top of the plan
-		auto window = make_unique<PhysicalWindow>(types, move(select_list), op.estimated_cardinality);
```

Comment:
Preprocessing `remaining` to put the streaming candidates at the end would result in them being emitted last.

Reply:
Done!